schema.json

```diff
@@ -10,7 +10,7 @@
    ],
    "countryCode": "FR",
    "homepage": "https://github.com/etalab/schema-impact-service-numerique-public",
-   "path": "https://github.com/etalab/schema-impact-service-public-numerique/raw/v0.1/schema.json",
+   "path": "https://github.com/etalab/schema-impact-service-public-numerique/blob/master/schema.json",
```

Comment:
Shouldn't it be a raw path? Ex: https://raw.githubusercontent.com/etalab/schema-impact-service-public-numerique/master/schema.json

Reply:
True, thanks!